pages/common/traceroute.md

```diff
@@ -17,3 +17,7 @@
 - Specify number of queries per hop:
 
 `traceroute -q 5 {{host}}`
+
+- Specify size in bytes of datagram:
+
+`traceroute {{host}} {{size<=32768}}`
```

Comment:
This is misleading. I almost thought that "size<=" was part of the syntax. Whereas its just the number of bytes.

Also, since the default is 60 for IPv4 and 80 for IPv6, its better to give a smaller no. 
`traceroute {{host}} {{45}}` seems good enough.